app/models/sitewide_setting.rb

```diff
@@ -4,11 +4,22 @@ class SitewideSetting < ApplicationRecord
   validates :key, uniqueness: true # rubocop:disable Rails/UniqueValidationWithoutIndex
   validates_with SafeHtmlValidator
 
+  after_save :republish_ministers_if_reshuffle
+
   def human_status
     on ? "On" : "Off"
   end
 
   def name
     key.humanize
   end
+
+  def republish_ministers_if_reshuffle 
+    return unless key == "minister_reshuffle_mode"
+
+    payload = PublishingApi::MinistersIndexPresenter.new
+
+    Services.publishing_api.put_content(payload.content_id, payload.content)
+    Services.publishing_api.publish(payload.content_id, nil, locale: "en")
```

Comment:
```suggestion
    Services.publishing_api.publish(payload.content_id, locale: "en")
```

I think you can avoid the `nil` [because it's `update_type = nil`](https://github.com/alphagov/gds-api-adapters/blob/8c5224f9c331cf6a22d6065ef7f5de9fcbbbed20/lib/gds_api/publishing_api.rb#L123).

Reply:
👍